nilearn/experimental/conftest.py

```diff
@@ -60,6 +60,24 @@ def f(shape=()):
     return f
 
 
+@pytest.fixture
+def make_mini_mask(mini_mesh) -> Callable:
```

Comment:
Not sure why you are creating another mask fixture, but I'd say that when 'merging' the experimental module in the stable API we should clean up our test set up because I think we have 3 or 4 different surface mask fixture and I am not sure we need that many.

Fine for this PR though.

Reply:
Yes it does look like we need better fixtures for creating sample surface images.

I was super confused as well because I could not import any of the existing fixtures for making sample surface masks. 

Then I noticed the same thing with sample surface images, and there was an existing `make_mini_img` being used repeatedly. So I created a similar one for masks.

Reply:
yeah the first thing we will most likely do when merging the new API will be clean up the tests in general: also because some fixture names will need to be clarified to avoid clashing with the volume fixture.